Kitodo/src/test/java/test/DBConnectionTestIT.java

```diff
@@ -11,27 +11,31 @@
 
 package test;
 
+import java.io.IOException;
+
 import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.Test;
-import org.kitodo.data.database.beans.Process;
+import org.kitodo.MockDatabase;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.data.elasticsearch.exceptions.CustomResponseException;
 import org.kitodo.services.ServiceManager;
-
 public class DBConnectionTestIT {
 
+    @BeforeClass
+    public static void prepareDatabase() throws DAOException, IOException, CustomResponseException {
+        MockDatabase.insertProcessesFull();
+    }
+
     @Test
     public void test() throws Exception {
-
-        Process test = new Process();
-        test.setTitle("First process");
         ServiceManager serviceManager = new ServiceManager();
-        serviceManager.getProcessService().save(test);
 
         long counted = serviceManager.getProcessService().count("from Process");
         Assert.assertNotNull("No Process found", counted);
         Assert.assertEquals(4, counted);
 
         String title = serviceManager.getProcessService().find(1).getTitle();
-        Assert.assertEquals("First process", title);
-
+        Assert.assertEquals("DBConnectionTest", title);
```

Comment:
This test could not work, as you are search for first process but your test data is inserted as 4th.